### ISSUE TITLE
No calendars shown

### ISSUE THREAD

**Issue by zagatta** [kewisch/gdata-provider#30]
Thundebird 60.9.0 (64-bit)
Provider for Google Calendar 4.4.2
Elementary OS 5.0 Juno

After entering my google account details, in the window to select the calendars to synchronize, there are NO calendars shown.

![Screenshot from 2019-11-12 10 34 08](https://user-images.githubusercontent.com/31660718/68659946-896ab280-0538-11ea-9088-46134f5ab8f7.png)

**Comment by kvap** [comment]
Experiencing the same.

**Comment by pablo-na** [comment]
Same here

**Comment by pablo-na** [comment]
I have the same versions of Thunderbird and Provider for Google Calendar in ubuntu and I'm experiencing the same issue. After allowing access to my Google Calendar, I went to my google account settings and checked whether Provider for Google Calendar or Lightning appeared and they don't. :( No calendars are shown either.

**Comment by rmorr0** [comment]
Same issue.

**Comment by Rinaldo-60** [comment]
Which version of gprovider are you using?
See if you set ThunderBird config setting "general.useragent.compatMode.firefox" to true, restart TB and check if you can add the calendar(s).

Might be the same issue as: https://github.com/kewisch/gdata-provider/issues/28

**Comment by pablo-na** [comment]
Worked like magic, thank you very much.

**Comment by kkrieger85** [comment]
Faced the same issue. 
@Rinaldo-60 's solution works fine.

**Comment by skrevier** [comment]
> Which version of gprovider are you using?
> See if you set ThunderBird config setting "general.useragent.compatMode.firefox" to true, restart TB and check if you can add the calendar(s).
> 
> Might be the same issue as: #28

Worked! Awesome! Thanks!

**Comment by gshpinneberg** [comment]
Version 60.9.1
Language: German

I can't find 
ThunderBird config setting "general.useragent.compatMode.firefox"

Any hints where that could be?
Thx!

**Comment by Rinaldo-60** [comment]
I am already on 68.
Have a look at Edit, Preferences, Advanced and there you find the "config editor"
But I don't know if it is on the same spot in the "older" version 60.

**Comment by gshpinneberg** [comment]
I updated my version. Works perfectly now. Thanks a lot!

**Comment by habamax** [comment]
Same issue, win10, Latest gdata-provider from extensions

![image](https://user-images.githubusercontent.com/234774/80190220-9bd73500-861c-11ea-9ac9-15a0d3dc51f9.png)


Works for all google calendars I have except one, most important for me :(

![image](https://user-images.githubusercontent.com/234774/80190097-5f0b3e00-861c-11ea-9157-36ad20b2e39d.png)

I can see an error after this window is opened:

```
[Exception... "[object Object]"  nsresult: "0x804a0402 (<unknown>)"  location: "JS frame :: resource://gdata-provider/modules/gdataRequest.jsm :: fail :: line 244"  data: no] gdataRequest.jsm:244:18
    fail resource://gdata-provider/modules/gdataRequest.jsm:244
    onStreamComplete resource://gdata-provider/modules/gdataRequest.jsm:494
    openCalendarWizard resource://calendar/modules/utils/calWindowUtils.jsm:26
    doCommand chrome://calendar/content/calendar-common-sets.js:375
    goDoCommand chrome://global/content/globalOverlay.js:101
    oncommand chrome://messenger/content/messenger.xul:1
```

![image](https://user-images.githubusercontent.com/234774/80190411-f07ab000-861c-11ea-8be6-aa6c0e1bbe49.png)

**Comment by kewisch** [comment]
The statusPrivacy thing I saw somewhere else, and I believe this if fixed in the version I posted in #77 for Thunderbird 78. Looks like all other issues here are resolved!
